lib/vsc/administration/user.py

```diff
@@ -305,16 +318,23 @@ def _create_user_dir(self, grouping_f, path_f, storage_name):
         @type grouping: function that yields the grouping path for the location.
         @type path: function that yields the actual path for the location.
         """
+        try:
```

Comment:
make a method for this, return fs_backend and storage

Reply:
totally changed to use the new `StorageOperator` class in vsc-filesystems. See https://github.com/hpcugent/vsc-administration/pull/134/commits/557c3681738c7694cc80cfbef58535b6d0325e83